vignettes/howtogenefilter.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+output:
+  html_document: default
+  pdf_document: default
+---
```

Comment:
Your YAML header should look similar to items 14 and 15 in the [contribution guide](https://bioconductor.github.io/sweave2rmd/articles/contribute.html#create-the-r-markdown--rmd-vignette-and-remove-the-sweave-vignette--rnw). Please also see [AuthoringRmdVignettes.Rmd](https://github.com/Bioconductor/BiocStyle/blob/devel/vignettes/AuthoringRmdVignettes.Rmd) and its [HTML](https://bioconductor.org/packages/3.17/bioc/vignettes/BiocStyle/inst/doc/AuthoringRmdVignettes.html) as an example.